health/health.go

```diff
@@ -32,21 +35,35 @@ var (
 	ipnWantRunning          bool
 )
 
+// ErrorKey is an overall category for which an error is being reported.
+type ErrorKey string
+
+const (
+	KeyOverall = ErrorKey("overall")
```

Comment:
I find this surprisingly confusing. Maybe there's some obvious way of thinking about this that makes it make sense?

Maybe it needs some docs?

Reply:
Whoops, my docs on ErrorKey itself also used word "overall" by accident, without realizing it conflicted with this "overall", which is different. I'll reword there and add docs here.

Reply:
Done. PTAL.